aws/cloudformation/components/ami.yml.erb

```diff
@@ -112,18 +112,17 @@ frontend_device_name ||= '/dev/sda1'
     LaunchConfiguration: {Ref: 'FrontendLaunchConfig'}
   %>
   Frontends:
-    DependsOn: [ASGCount]
```

Comment:
Did the linter say why this is not necessary?

Reply:
Yes, though I could be convinced to leave it for extra clarity and add an exception to the linter.

The presence of `!GetAtt ASGCount.<property>` later in the resource is enough to define the dependency. Cloudformation builds the dependency tree by looking at `!GetAtt` and `!Ref`, so you only really need `DependsOn` if for some reason that's not sufficient.

Reply:
Cool, thanks! I think that makes enough sense that an exception isn't necessary